packages/atchops/include/atchops/aes.h

```diff
@@ -2,10 +2,11 @@
 #ifndef ATCHOPS_AES_H
 #define ATCHOPS_AES_H
 
-typedef enum atchops_aes_keysize {
+enum atchops_aes_size {
```

Comment:
I think it's cool that we're dropping the typedef, but I also don't see the justification for the change.

MbedTLS's hash enums are a typedef, which we take as an argument in rsa_sign. Maybe it would be good to remain consistent?

```c
typedef enum {
    MBEDTLS_MD_NONE=0,    /**< None. */
    MBEDTLS_MD_MD5=0x03,       /**< The MD5 message digest. */
    MBEDTLS_MD_RIPEMD160=0x04, /**< The RIPEMD-160 message digest. */
    MBEDTLS_MD_SHA1=0x05,      /**< The SHA-1 message digest. */
    MBEDTLS_MD_SHA224=0x08,    /**< The SHA-224 message digest. */
    MBEDTLS_MD_SHA256=0x09,    /**< The SHA-256 message digest. */
    MBEDTLS_MD_SHA384=0x0a,    /**< The SHA-384 message digest. */
    MBEDTLS_MD_SHA512=0x0b,    /**< The SHA-512 message digest. */
    MBEDTLS_MD_SHA3_224=0x10,  /**< The SHA3-224 message digest. */
    MBEDTLS_MD_SHA3_256=0x11,  /**< The SHA3-256 message digest. */
    MBEDTLS_MD_SHA3_384=0x12,  /**< The SHA3-384 message digest. */
    MBEDTLS_MD_SHA3_512=0x13,  /**< The SHA3-512 message digest. */
} mbedtls_md_type_t;
```

Reply:
I dropped the typedef because they really shouldn't be passing around the type.